### DISCUSSION TITLE
OFFICE HOURS SCHEDULED: Tuesdays 9-11, Tuesdays 4-6, Fridays 1-3

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#189] [Announcements]
@earthlab-education/ea-students @earthlab-education/ea-instructors It seems like we need some office hours ASAP! Sorry we got a late start because of the holiday weekend - sounds like there was a lot of questions (which is totally expected - especially this GitHub stuff we’ll be working on throughout the semester)

Hours will be in the class zoom at:

Tuesdays 9-11
Tuesdays 4-6
Fridays 1-3

If you need to meet another time - please try the discussion board and email first, but you are welcome to [schedule a meeting with me using calendly](https://calendly.com/elsa-culler). I’ll be available Tuesdays-Saturdays so we have some weekend time.

**Comment by nquarder** [comment]
looks good @eculler !
